.github/workflows/site.yml

```diff
@@ -1,39 +1,39 @@
-# Simple workflow for deploying static content to GitHub Pages
```

Comment:
We can remove it entirely instead of commenting it out. It is in git history so nothing to worry about if we want to restore it later.